dysnix/ai-scale-saver/templates/_default.tpl

```diff
@@ -0,0 +1,60 @@
+{{- define "default.service.configs" -}}
+debugMode: true
+profiling:
+  enabled: true
+monitoring:
+  enabled: true
+single:
+  enabled: true
+  host: 0.0.0.0
+  port: 8097
+  name: pprof/monitoring server
+  concurrency: 100000
+  buffer:
+    readBufferSize: 4MiB
+    writeBufferSize: 4MiB
+  tcpKeepalive:
+    enabled: true
+    period: 1s
+  httptransport:
+    maxIdleConnDuration: 15s
+    readTimeout: 7s
+    writeTimeout: 7s
+elasticsearch:
+  client:
+    address: http://localhost:9200
+    username: ""
+    password: ""
+    maxRetries: 7
+    transport:
+      maxIdleConnDuration: 1m
+      readTimeout: 7s
+      writeTimeout: 7s
+  processor:
+    workersCount: 10
+    flushBytes: 4KiB
+    flushInterval: 30s
+    indexTemplatePatternName: ai-scale
+    batchTimeout: 1m
+    bulkTimeout: 10s
+grpc:
+  enabled: true
+  useReflection: true
+  compression:
+    enabled: false
+    type: Zstd
```

Comment:
is this case insensitive?

Reply:
yes. It's enum with custom marshaller/unmarshaller.